airflow-core/src/airflow/secrets/local_filesystem.py

```diff
@@ -278,6 +278,26 @@ def load_connections_dict(file_path: str) -> dict[str, Any]:
     return connection_by_conn_id
 
 
+def load_configs_dict(file_path: str) -> dict[str, str]:
+    """
+    Load configs from a text file.
+
+    ``JSON``, `YAML` and ``.env`` files are supported.
+
+    :param file_path: The location of the file that will be processed.
+    :return: A dictionary where the key contains a config name and the value contains the config value.
+    """
+    log.debug("Loading configs from a text file")
```

Comment:
nit: logging can be including the filename
```suggestion
    log.debug("Loading configs from text file %s", file_path)
```